core/json/.htaccess

```diff
@@ -0,0 +1,5 @@
+<Files "variables.json">
+    Order Deny,Allow
+    Deny from all
+    Allow from 127.0.0.1
+</Files>
```

Comment:
Please use Apache 2.4 access control: http://httpd.apache.org/docs/current/howto/access.html
Or make an if module ... and include both.